package.json

```diff
@@ -11,7 +11,7 @@
   "engines": {
     "node": ">=8.0.0 <11",
     "npm": ">=5.0.0",
-    "mongodb": "~3.4"
+    "mongodb": ">=3.0 <=4.0"
```

Comment:
This may need a second check. Not sure what should be here. https://mongoosejs.com/docs/compatibility.html

Reply:
Mongo tends to break between minor point releases (3.4 vs 3.5) so I'd keep this as strict as possible for now. Let's manually test and target 3.6

Reply:
I'm currently testing against mongodb v 4.0.0.

The #580 queries work. Tested manually.

Reply:
Excellent! :thumbsup: I'll do some testing as well and attempt to upgrade production to MongoDB v4 as well.